university-service/src/testFixtures/kotlin/validation/ValidationTestFixture.kt

```diff
@@ -0,0 +1,12 @@
+package validation
+
+import javax.validation.Validation
+import javax.validation.Validator
+
+class ValidationTestFixture {
+
+    companion object {
```

Comment:
please look at this [comment](https://github.com/afanasievM/rest-project/pull/9#discussion_r1318225598).

We can simplify the class by using `object`

Reply:
ok. 
I don't know why I miss it(